src/background/extension.ts

```diff
@@ -410,11 +410,15 @@ style && style.parentElement.removeChild(style);
          * installed in system.
          */
         protected getFontList(onReturned: (fonts: string[]) => void) {
-            chrome.fontSettings.getFontList((res) => {
-                // id or name?
-                var fonts = res.map((r) => r.fontId);
-                onReturned(fonts);
-            });
+            if(chrome.fontSettings != undefined){
```

Comment:
Replace if/else with if/return, so that we can just simply remove this when fonts will be supported and leave a Todo comment.